### DISCUSSION TITLE
Welcome to resonance-market Discussions!

### DISCUSSION THREAD

**Discussion by yjl9903** [yjl9903/resonance-market#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Velsun3366** [comment]
你真的好强，单人开发速度不是我能比的，UI层我倒是没压力，有关SQLite、drizzle，类似后端的node包的使用说明都够我喝一壶的了，而且这个还是nuxt。汗流浃背-_-||
